indexeddb-files/README.md

```diff
@@ -0,0 +1,35 @@
+# indexeddb-files
+
+This add-on shows how to read and save files using the IndexedDB API and the sub-set of the "File and Directory Entries API" supported on Firefox.
+
+## What it does ##
+
+The extension includes:
+
+* a browser action which opens a new extension tab
+* 3 examples that can be loaded by the extension tab opened by clicking the browser action.
+
+## What it shows ##
+
+- **Reading files from a local directory tree**: which shows how to use the sub-set of the "File and Directory API" supported on Firefox to read a directory tree instead of single files
+- **Save files into IndexedDB**: which shows how to use the IndexedDB files to save and read files from indexedDB (and how to extract of the files as a single zip files)
```

Comment:
"extract of the files as a single zip files" -> "download the files as a single zip file".

You could also say that this shows how to save the contents of a local directory tree into indexedDB.